contracts/RelayHub.sol

```diff
@@ -217,7 +222,7 @@ contract RelayHub is IRelayHub {
     {
         if (!vars.success) {
             //Failure cases where the PM doesn't pay
-            if ( (vars.innerGasUsed < vars.gasLimits.acceptanceBudget ) && (
+            if ( (vars.innerGasUsed < vars.gasAndDataLimits.acceptanceBudget ) && (
                     vars.status == RelayCallStatus.RejectedByPreRelayed ||
```

Comment:
This was supposed to be changed so that `RejectedByPreRelayed` is considered top priority rule.

Reply:
Not in this PR